config.gemspec

```diff
@@ -25,11 +25,9 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'activesupport',  '>= 3.0'
   s.add_dependency 'deep_merge',     '~> 1.2',   '>= 1.2.1'
+  
   s.add_dependency 'dry-validation', '~> 0.12', '>= 0.12.2', '< 1.0.0' if RUBY_VERSION >= '2.2'
-
-  if RUBY_VERSION >= '2.1' && RUBY_VERSION < '2.2'
-    s.add_dependency 'dry-validation', '~> 0.10', '>= 0.10.7', '< 1.0.0'
-  end
+  s.add_dependency 'dry-validation', '~> 0.10', '>= 0.10.7', '< 1.0.0' if RUBY_VERSION >= '2.1' && RUBY_VERSION < '2.2'
```

Comment:
Code climate complained that this line was too long. This is how I had it originally.

Reply:
Does not seem to complain anymore?